### PR TITLE
Fixes in showcases and documentation UI

### PR DESCRIPTION
**Pull request by IgorWilbert**
For #2773

#### Change List
- A typo in showcases "Autonomous Visuaization System" was fixed;
- In showcases, links from menu open in a new tab, but links in image or text open in the current tab. All  of them now open in a new tab;

### CODE REVIEW

File: website/src/components/header.js [hunk 1]
```diff
@@ -46,6 +46,9 @@ export default class Header extends Component {
             <a href={FRAMEWORK_GITHUB_URL}>
               Github<i className="icon icon-github" />
             </a>
+            <a href="https://github.com/mapbox/mapbox-gl-js">
```
Comment (Pessimistress) [inline website/src/components/header.js]: Can you remove this? deck.gl is Mapbox compatible, but it does not have to be used with Mapbox. We do link to the Mapbox repo in http://uber.github.io/react-map-gl/

File: website/src/static/markdown/showcase.md [hunk 2]
```diff
@@ -1,91 +1,91 @@
 <div>
   <div class="thumb">
-    <div class="bg-black" data-title="Autonomous Visuaization System" >
-      <a href="http://streetscape.gl/">
+    <div class="bg-black" data-title="Autonomous Visualization System" >
+      <a href="http://streetscape.gl/" target="newtab">
```
Comment (Pessimistress) [inline website/src/static/markdown/showcase.md]: Use `target="_blank" rel="noopener noreferrer"`

File: website/src/stylesheets/_gallery.scss [hunk 3]
```diff
@@ -12,7 +12,7 @@
 }
 .toc {
   min-width: 240px;
-  max-width: 320px;
+  max-width: 360px;
```
Comment (Pessimistress) [inline website/src/stylesheets/_gallery.scss]: Let's leave this out for now. We will drop the (experimental) tag from the title in the next release.

**Comment by IgorWilbert** [comment]
@Pessimistress all done, thanks!

**Pull request merged**